src/contractTest/java/uk/gov/hmcts/reform/roleassignment/RoleAssignmentProviderTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.roleassignment;
+
+import au.com.dius.pact.provider.junit5.PactVerificationContext;
+import au.com.dius.pact.provider.junit5.PactVerificationInvocationContextProvider;
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import au.com.dius.pact.provider.junitsupport.loader.PactBroker;
+import au.com.dius.pact.provider.spring.junit5.MockMvcTestTarget;
+import org.json.JSONException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.roleassignment.controller.endpoints.GetAssignmentController;
+import uk.gov.hmcts.reform.roleassignment.domain.service.getroles.RetrieveRoleAssignmentOrchestrator;
+import org.springframework.context.annotation.Import;
+import java.io.IOException;
+
+@ExtendWith(SpringExtension.class)
+@Provider("am_role_assignment_service")
+@PactBroker(scheme = "http", host = "localhost", port = "9292")
+@Import(RoleAssignmentProviderTestConfiguration.class)
```

Comment:
does pactbroker need to be hardcoded to localhost?

Reply:
It needs to connect to HMCTS broker in order for run in pipeline.

Reply:
This is updated for higher env.